examples/cube/src/main.rs

```diff
@@ -114,33 +114,35 @@ fn main() {
     let object = rend3::types::Object {
         mesh: mesh_handle,
         material: material_handle,
-        transform: glam::Mat4::IDENTITY,
+        transform: Mat4::IDENTITY,
     };
     // Creating an object will hold onto both the mesh and the material
     // even if they are deleted.
     //
     // We need to keep the object handle alive.
     let _object_handle = renderer.add_object(object);
 
+    let view_location = Vec3::new(3.0, 3.0, -5.0);
+    let view_direction: Vec3 = (Mat3A::from_euler(glam::EulerRot::YXZ, -0.55, 0.5, 0.0) * Vec3A::Z).into();
+    let view = Mat4::look_at_lh(view_location, view_location + view_direction, Vec3::Y);
```

Comment:
We should have a `Camera::from_euler` to simplify this.

Reply:
Yeeah, I was thinking that you might want some helper, but wasn't sure what shade of bikeshed.
I suppose you don't want me to just replicate the glam from_euler.. do you perhaps want something like `from_yaw_pitch_and_location(yaw,pitch,location)`? just from_euler for those params seems maybe a bit confusing

Reply:
Heh, yeah. `from_yaw_pitch_roll_location` should work witch take the args in that order. If you're up to it, it would be nice to also have a `from_quat_location` to add @jamen's use case. Each of these can take a CameraProjection as the first argument.

Sound reasonable?